FrankenDrift.GlkRunner/FrankenDrift.GlkRunner/GlkHtmlWin.cs

```diff
@@ -107,50 +107,48 @@ public void Clear()
             GlkApi.glk_window_clear(glkwin_handle);
         }
 
-        internal unsafe string GetLineInput()
+        internal string GetLineInput()
         {
             if (IsWaiting)
                 throw new ConcurrentEventException("Too many input events requested");
             IsWaiting = true;
             const uint capacity = 256;
-            var cmdToBe = new uint[capacity];
-            fixed (uint* buf = cmdToBe)
+            Memory<UInt32> buf = new uint[capacity];
```

Comment:
It looks like I changed lots here, but this line and `var buf_span = buf.Span;` below are the only substantial changes, everything else is just whitespace from removing the `fixed` indentation.